docs/tutorials/motor_noise_debugging.rst

```diff
@@ -194,3 +194,79 @@ vibration resistant features include, but are not limited to the following:
 
 Additionally, consider making redundant ground connections through a fuse or polymeric positive temperature coefficient device (PTC) to prevent high current flows through 
 a carbon fiber frame.
+
+Grounding Vetiq Motors Frames
+#####################################################
+
+Vertiq's Pro Kit motors as well as many modular motors in the 40, 60, and 81 diameter sizes are shipped with a user accessible, nonpopulated component footprint. 
+This component footprint provides a convenient method for connecting the motor’s frame to the V- electrical ground through a mounting screw. Illustrations of the 40, 60, 
+and 81 sized motors and their connector boards will be provided below. The nonpopulated footprint’s pads and the mounting screws are labeled:
+
+	1. Solderpad connected to V-/ground 
+	2. Solderpad connected to grounding screw 
+	3. Grounding screw 
+	4. Mounting screw. 
+
+NOTE: Component designator labels (F1, R1, etc) may not appear on connector board silkscreen.
```

Comment:
There is a more RTD-y way to add notes like this. If you do this (example stolen from elsewhere):
![image](https://github.com/user-attachments/assets/77e22062-0ce5-491d-a34e-760137796cd4)

Then it will look like this on RTD:
![image](https://github.com/user-attachments/assets/1fc92946-07b6-4417-8a00-f24ac1bcde21)